Ziptastic/src/main/java/Ziptastic_Object/Ziptastic.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Ziptastic_Object;
+
+import java.io.IOException;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+
+public class Ziptastic {
+    
+    private static String url = "https://zip.getziptastic.com/v3/US/";
+    private static String apiKey;
+    
+    public Ziptastic(String key) {
+       this.apiKey = key; 
+    }
+    
+    
+    public static String ForwardGeocoding(int zipcode) throws IOException{

```

Comment:
1. ForwardGeocode() works for me.
2. is there a way to do it so you don't have to duplicate the http request code as well?